### PR TITLE
i18n(de): update `recipes/add-yaml-support.mdx` 

### PR DESCRIPTION
**Pull request by trueberryless**
#### Description (required)

- update `recipes/add-yaml-support.mdx` 
(one of the few files that survived #9240)
- change translation of `Rezept` (recipe) to `Beispiel` (example) because it fits better in German

#### Related issues & labels (optional)

- #4666
- #4805
- #5305
- #5846
- #8167

### CODE REVIEW

**Review by sarah11918 — DISMISSED** [review]
Blocking so this does not get accidentally merged while we are in a code freeze for v4 docs.

**Comment by sarah11918** [comment]
OK @trueberryless ! V5 is out, so I'm taking off the block, but please do update your branch because the site has changed A LOT since this PR was made. 😅  I'm sorry if there might be conflicts to deal with!

**Comment by trueberryless** [comment]
Thank you very much and sry for causing some trouble shortly before release...

File: src/content/docs/de/recipes/add-yaml-support.mdx
Comment (yanthomasdev) [inline src/content/docs/de/recipes/add-yaml-support.mdx:1]: The comments on line 59-61 haven't been translated, could you take care of it @trueberryless?

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**